shared/components/SpeakersList/index.js

```diff
@@ -0,0 +1,93 @@
+/* eslint-disable react/no-danger */
+
+import React, { PropTypes } from 'react';
+
+export const speakerType = PropTypes.shape({
+  id: PropTypes.string,
+  name: PropTypes.string,
+  company: PropTypes.string,
+  twitterHandle: PropTypes.string,
+  githubHandle: PropTypes.string,
+  blogURL: PropTypes.string,
+  imageURL: PropTypes.string,
+});
+
+export const speakerBio = (bio) => {
+  const fullBio = [];
+  if (bio
+    && Array.isArray(bio)
+    && bio.length) {
+    bio.forEach((paragraph) => {
+      if (paragraph.text) {
+        fullBio.push(paragraph.text);
+      }
+    });
+  }
+  return fullBio.join('\r\n\r\n');
+};
+
+export const speakerSocialLinks = (speaker) => {
+  const githubLink = speaker.githubHandle ?
+    (<a href={`https://github.com/${speaker.githubHandle}`}><span className="icon-github" /></a>)
+    : null;
+  const twitterLink = speaker.blogURL ?
+    (<a href={speaker.blogURL}><span className="icon-link" /></a>)
+    : null;
+  const blogLink = speaker.twitterHandle ?
+    (<a href={`https://twitter.com/${speaker.twitterHandle}`}><span className="icon-twitter" /></a>)
+    : null;
```

Comment:
Also it should use the `ExternalLink` component, not an `a`. I notice that this is not done in the `Speaker` element either :(